rfc-139-enable-http3.md

```diff
@@ -0,0 +1,135 @@
+
+![Difference between TCP and QUIC when downloading assets.](rfc-139/QUIC-illustration-final.jpg)
+
+In the illustration above, if packet 2 is lost (using HTTP/2), packets 3-8 are blocked until retransmission occurs. Using QUIC if packet 2 is lost only packets 1 and 3 are blocked since they exist in the same stream. The other streams can progress as normal since they are fully independent. QUIC doesn't require in-order packet transmission across the whole connection, but it does still retain ordering within each resource stream.
+
+## The effect of packet loss
+
+Using WebPageTest we can roughly simulate the effect that packet loss has on performance for our users. For testing I picking a fairly content heavy page like the [past Prime Ministers](https://www.gov.uk/government/history/past-prime-ministers) page and a low spec Moto G4 Android device. For the connection speed I've picked on one of the worse case scenarios from the [MLab data](https://docs.google.com/spreadsheets/d/1UTRucwLY2VC4944-cKunL9yUPJkjDKXDdIPk6GY3610/edit#gid=0). I'm emulating the connection the village of Colintraive has, which reportedly has a 450/230 Kbps connection speed. This is a connection speed equivalent to a 2G or slow 3G connection.
+
+### Visual filmstrip
+Below we can see a comparison of the page load under different packet loss conditions using HTTP/2. The top is the baseline, with 0% packet loss, progressing all the way down to 3% packet loss:
+
+![Comparing the visual filmstrip of the effect of packet loss on a page load](rfc-139/visual-filmstrip.png)
+
+As you can see, the visually complete metric sits at 9.5 seconds with no packet loss, but jumps to 13.5 seconds with 3% packet loss. This is a 42% increase.
+
+### Visual progress graph
+The visual progress graph clearly shows the effect packet loss has on page load times. As the packet loss increases, the page load slows down:
+
+![The visual progress graph comparing the percentage completeness of a page over time.](rfc-139/visual-progress.png)
+
+HTTP/3 + QUIC is designed to help improve the page load speed for users experiencing these connection conditions in the real world. 
+
+### Connection quality measurement
+Is it possible to measure the connection quality that our users are experiencing when visiting GOV.UK. Fastly provide a [range of Varnish Configuration Language (VCL) options](https://developer.fastly.com/reference/vcl/variables/client-connection/) that we can use to monitor the quality of user connections should we wish to capture this data:
```

Comment:
I assume this is a statement rather than a question?

```suggestion
It is possible to measure the connection quality that our users are experiencing when visiting GOV.UK. Fastly provide a [range of Varnish Configuration Language (VCL) options](https://developer.fastly.com/reference/vcl/variables/client-connection/) that we can use to monitor the quality of user connections should we wish to capture this data:
```

I'm not quite sure what the aim of this section is, though. Are you proposing that as part of this RFC, we _should_ leverage some VCL to start monitoring the packet loss of our users?